### DISCUSSION TITLE
plot age and finding coins question.

### DISCUSSION THREAD

**Discussion by katmai** [Chia-Network/chia-blockchain#5823] [General]
so i have a question. as plots become older and older, do they all get the same shot at winning blocks as the newer created plots?

**Comment by chiabug** [comment]
In theory yes but who is determining the latest challenge hash?  The challenge hash (effectively the lottery ticket number) is important.  If it is being skewed in any manner then it could be swayed away from older plots.  How are these numbers actually being picked and disseminated?

The bigger issue is the duplication rate of the lottery numbers in your plot.  As the netspace increases the duplication of your numbers increases.  This decreases your ability to answer a challenge hash and win that challenge with the dominant proof.  The proof coming from a well connected pool will likely override the proof submitted from a small farmer connected to random peers because of the lag time across the global network.
